superset/assets/src/dashboard/stylesheets/filter-indicator.less

```diff
@@ -0,0 +1,97 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+.dashboard-filter-indicators-container {
+  position: absolute;
+  z-index: 10;
+  right: -20px;
+  top: 40px;
+  width: 20px;
+  height: 125px;
+
+  .indicator-container {
+    position: relative;
+    margin-bottom: 4px;
+    margin-left: 30px;
+    left: -30px;
+  }
+
+  .filter-indicator,
+  .filter-indicator-group {
+    width: 3px;
+    height: 20px;
+    overflow: hidden;
+    display: flex;
+    background-color: #bababa;
+    transition: all 0.3s;
```

Comment:
not a big fan of transition all, can we explicitly call out what styles should be transitioned over?

Reply:
fixed. only width property.